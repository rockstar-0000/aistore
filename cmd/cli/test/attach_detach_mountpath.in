ais storage mountpath attach // FAIL
ais storage mountpath detach // FAIL
ais storage mountpath attach something // FAIL "invalid key=value pair "something""
ais storage mountpath detach something // FAIL "invalid key=value pair "something""
ais storage mountpath attach $RANDOM_TARGET=/nonexisting // FAIL "does not exist"
ais storage mountpath detach $RANDOM_TARGET=/nonexisting // FAIL "does not exist"

ais storage mountpath detach $RANDOM_TARGET=$RANDOM_MOUNTPATH
ais storage mountpath attach $RANDOM_TARGET=$RANDOM_MOUNTPATH
