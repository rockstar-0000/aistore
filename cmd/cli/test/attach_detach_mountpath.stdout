^.*$RANDOM_TARGET.*now detached.*$
^.*$RANDOM_TARGET.*now attached.*$
